src/Stratis.Features.SQLiteWalletRepository/Tables/HDWallet.cs

```diff
@@ -111,7 +111,7 @@ internal class HeightHashPair
             internal string Hash { get; set; }
         }
 
-        internal static void AdvanceTip(SQLiteConnection conn, HDWallet wallet, ChainedHeader newTip, uint256 prevTipHash)
+        internal static void AdvanceTip(DBConnection conn, HDWallet wallet, ChainedHeader newTip, uint256 prevTipHash)
```

Comment:
How come this change?

Reply:
After the change the `DBConnection.Execute` method, which includes logging, will be used.